app/services/notifier.rb

```diff
@@ -22,13 +22,13 @@ def self.publish(event_name, **payload)
 
   def self.subscribe_mailer(event_name:, mailer_class:, mailer_method:)
     ActiveSupport::Notifications.subscribe(event_name) do |event|
-      mailer_class.with(**event.payload).send(mailer_method).deliver_later
+      mailer_class.with(**event.payload).send(mailer_method).deliver_later if Settings.notifications.enabled
```

Comment:
This will allow us to turn off notifications in some tests where they prove pesky.

Reply:
Some of the emails requiring hitting SDR to get the cocina. In test, SDR doesn't know anything about the test objects so an error is raised. This happens in the spec for DepositCompleteJob.

SDR could be mocked out in such tests, but since the email is tangential to the thing under test it seems cleaner to just turn off notifications.

Reply:
@justinlittman Ugh. Yes, agree.